packages/vscode-extension/src/officeChat/commands/create/officeCreateCommandHandler.ts

```diff
@@ -40,8 +41,8 @@ export default async function officeCreateCommandHandler(
 
   if (request.prompt.trim() === "") {
     response.markdown(localize("teamstoolkit.chatParticipants.officeAddIn.create.noPromptAnswer"));
-
-    officeChatTelemetryData.markComplete();
+    officeChatTelemetryData.setBlockReason("Empty Input");
```

Comment:
Should the 'Empty Input' be set in an Enum?

Reply:
yes I missed it. will fix, thanks